### ISSUE TITLE
[Pods 2.8.1] Multi Select with Relationship data (ACT) does not save selected items

### ISSUE THREAD

**Issue by iNalgiev** [pods-framework/pods#6216]
## Description

As of Pods 2.8, I'm running into the following issue. 
I'm using primarily ACT tables with relationships. It seems that if you select multiple options in a Multi Select field, it will not save all the selected options.

However, when using checkboxes (as input type), it seems to work fine. See screenshot below.

## Testing instructions

1. Create a ACT
2. Add a field with a relationship to another ACT (with lots of data)
3. Edit the created field
4. In Relationship Options under Input Type select Multi Select (basic selection)
5. When editing the ACT, select multiple options by CTRL + left mouse clicking in the field.

## Expected behavior

All the multiple selected options should be saved.

## Screenshots / screencast

![image](https://user-images.githubusercontent.com/7498667/138456008-27d2178e-6445-479e-b4f5-dd7c4c9c0c71.png)

## Possible workaround

Temporarily change the Input Type to Checkboxes. 

## Pods version

Pods 2.8.1
WP 5.8.1

**Comment by sc0ttkclark** [comment]
Following up here --

Here's the screencast of me reproducing it this weekend: https://share.skc.dev/7KuAEOGB

JSON package: https://share.skc.dev/WnukkYvm

**Comment by sc0ttkclark** [comment]
Seems related to our SimpleSelect input handling, perhaps because we build the options out manually instead of dynamically.

I confirmed this happens with both Single `dropdown` / Multiple `multiselect`

**Comment by sc0ttkclark** [comment]
There's a fix for this in the `release/2.8.9` branch now: https://github.com/pods-framework/pods/archive/refs/heads/release/2.8.9.zip

**Comment by iNalgiev** [comment]
Hi @sc0ttkclark 
Thank you for the update. 

I've installed version 2.8.9 on my test site. Unfortunately the same issue appears. It does not save the multi selected options.

**Comment by sc0ttkclark** [comment]
@iNalgiev when you say it does not save, can you clarify whether you mean that it saves and closes the modal but does not add it to the list until refreshing the page?

**Comment by iNalgiev** [comment]
@sc0ttkclark I can confirm that it does not save the multi selected items (in the database) and also doesn't add the items to the list after refreshing the page.

**Comment by sc0ttkclark** [comment]
@iNalgiev the field seems to be working right now in my testing. I'm going to release 2.8.9 with the fixes so far and if we can uncover any other issues that can be reproduced then we can tackle those in 2.8.10 since this is already better than before based on my testing.

**Comment by iNalgiev** [comment]
@sc0ttkclark 
I think the issue described here: #6392  is very similar to what I'm experiencing. 
I'm curious to his (@myrmithron) results after updating to v2.8.10 (or v2.8.9).

**Comment by zrothauser** [comment]
I believe this should be fixed by PR #6481, which was meant as a fix for #6457. Relationships to ACTs edited in a modal were not working correctly since Pods 2.8, and this PR fixes that.

**Comment by iNalgiev** [comment]
Hi @sc0ttkclark & @zrothauser,
Thank you for the update.

I've installed Pods 2.8.11-a build locally and I'm still noticing the same issue. I'm afraid these fixes do not apply to my workflow. Below is a recording of the issue I'm facing: 
https://user-images.githubusercontent.com/7498667/159692312-d60002c2-9db9-4225-85fb-6b9930689d3a.mp4

Here's the export of the pods-package.json (if you want to reproduce it):
https://pastebin.com/aZaPajwn

**Comment by zrothauser** [comment]
Thanks for that @iNalgiev - I'll look into this further. @sc0ttkclark I think the title for the issue is inaccurate, based on the video - the issue with modals was separate from this one.

**Comment by sc0ttkclark** [comment]
@zrothauser this should likely be sending the values back as:

```
[pods_field_company] => Array
  (
    [0] => 3,
    [1] => 5,
    [2] => 6
  )
```

Can anything be done from the frontend side of the code here?

**Comment by sc0ttkclark** [comment]
Resolved via https://github.com/pods-framework/pods/pull/6498 and will be fixed in the 2.8.16 release
